### ISSUE TITLE
Multiple labels for one term

### ISSUE THREAD

**Issue by jahilton** [obophenotype/uberon#1466]
In composite-metazoan.owl, CL_0000134 has label 'mesenchymal stem cell' and label 'mesenchymal cell'
'mesenchymal cell' is also the label for CL_ 0008019
I do not see the same issue in http://purl.obolibrary.org/obo/cl.owl, which is why I thought to ask here first.
Please advise on how I can track down the root of this issue, or if I am misinterpreting the file in some way.

**Comment by cmungall** [comment]
I have tracked this down. One of the sources of composite-metazoan is http://obofoundry.org/ontology/ehdaa2

This unfortunately bundles an "old copy" of CL in with it

I have the ability to fix EHDAA2 which may be easier than fixing in this pipeline

**Comment by jahilton** [comment]
Just did a quick check of composite-metazoan.owl from the [June release](https://github.com/obophenotype/uberon/releases/tag/v2022-06-30) and the problem persists.
Lines 39685-39686 (under CL_0000134)
```
        <rdfs:label rdf:datatype="http://www.w3.org/2001/XMLSchema#string">mesenchymal cell</rdfs:label>
        <rdfs:label rdf:datatype="http://www.w3.org/2001/XMLSchema#string">mesenchymal stem cell</rdfs:label>
```

**Comment by matentzn** [comment]
@jahilton haha I did not expect you to answer on a 4 year old ticket, so that's a nice surprise. Ok I will add it to the next major round of review. Note that have frozen the composite PURL in time to a much older release because of a massive refactoring in October last year, but this here seems to preceded the migration.

- [ ] All SSAO's need to be base-ified before merging

**Comment by gouttegd** [comment]
Fixed in #3129.

(@matentzn Did you really tag this issue with `easy-fix`???? :D )
